modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -106,6 +118,25 @@ def format_representative(representative)
           }
         end
 
+        def validate_individual_poa_code!(poa_code)
```

Comment:
this validation logic will need to change once registration number is a required param